documentation/src/main/wiki/SourceCollector.md

```diff
@@ -6,7 +6,7 @@ Module name: **source**
 
 | ! Note    |
 | :-------- |
-| System is waiting up to 20 seconds before request is timed out. This parameter is configurable via OSGi configuration (`AET Source Collector Factory`). |
+| System is waiting up to 40 seconds before request is timed out. This parameter is configurable via OSGi configuration (`AET Source Collector Factory`). |
```

Comment:
Thank you! I don't ant to update defaults in this pull requests. If longer timeout is needed it should be done by OSGi configuration (conf files). I will revert this docs to 20 seconds..